keyboards/mechwild/sugarglider/wide_oled/f411/keyboard.json

```diff
@@ -1,3 +1,20 @@
 {
-    "development_board": "blackpill_f411"
+    "development_board": "blackpill_f411",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "rgblight": true,
+        "encoder": true,
+        "dip_switch": true,
+        "steno": true,
+        "oled": true,
+        "pointing_device": true,
+        # Enable Dynamic Tapping Term to control the Tap term for the Cirque Pad easily
+        "dynamic_tapping_term": true,
+        "wide_oled": true
```

Comment:
wide_oled isn't a standard feature, it probably should not be migrated.

Reply:
Could instead just add to the top of `keyboards/mechwild/sugarglider/post_rules.mk` 
```mk
WIDE_OLED_ENABLE ?= yes
```

Then the custom stuff is isolated to a single file

Reply:
Missed that part of the folder structure, yes it would.